src/sentry/grouping/strategies/configurations.py

```diff
@@ -1,6 +1,5 @@
 from sentry.grouping.strategies.base import (
     RISK_LEVEL_HIGH,
-    RISK_LEVEL_LOW,
```

Comment:
Should we keep `LOW` here, just so we have the complete set?

Reply:
I plan on cleaning up more code after this merges and remove things that are unnecessary.